src/core/cuda_wrapper.hpp

```diff
@@ -1,89 +0,0 @@
-/*
- * Copyright (C) 2010-2019 The ESPResSo project
- *
- * This file is part of ESPResSo.
- *
- * ESPResSo is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * ESPResSo is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-#ifndef CUDA_WRAPPER_HPP
-#define CUDA_WRAPPER_HPP
-
-#if defined(__CUDACC__)
-
-#include <cuda.h>
```

Comment:
@KaiSzuttor I'm replying here because your comment doesn't show up in the GitHub interface.

The `cuda_wrapper.hpp` only includes `cuda.h` once the HIP macros are removed. Do you want to keep it so that we can add CUDA macros in the future? Should we also keep the cufft and curand wrappers?

Reply:
i changed my mind :) it's probably better to remove all of those wrappers.